charts/cloudzero-agent/docs/releases/1.0.0.md

```diff
@@ -0,0 +1,122 @@
+region: null
+
+# -- UNCHANGED: CloudZero API key. Required if existingSecretName is null.
+apiKey: null
+
+# -- UNCHANGED: If set, the agent will use the API key in this Secret to authenticate with CloudZero.
+existingSecretName: null
+
+# -- NEW: Flag to deploy the Jetstack.io "cert-manager". Most environments will already have this deployed,
+#    so set this to "false" if applicable. Otherwise, enabling this to "true" is a quick way to get started.
+#    See the README for more information.
+cert-manager:
+  # -- DEFAULT: enabled.
+  enabled: true | false
+
+# -- NEW: Service Account used for the Insights Controller
+#    The account is required. If you already have an existing account, set the name in the field below.
+serviceAccount:
+  # -- DEFAULT: create the service account.
+  create: true | false
+  name: ""
+  annotations: {}
+
+# -- NEW: Label and Annotation Configuration
+insightsController:
+  # -- By default, a ValidatingAdmissionWebhook will be deployed to record all created labels and annotations.
+  enabled: true | false
+  labels:
+    # -- DEFAULT: enabled.
+    enabled: true | false
+    # -- This value MUST be set to a list of regular expressions used to gather labels from pods,
+    #    deployments, statefulsets, daemonsets, cronjobs, jobs, nodes, and namespaces.
+    patterns:
+      # List of Go-style regular expressions used to filter desired labels.
+      # Caution: The CloudZero system has a limit of 300 labels and annotations,
+      # so it is advisable to provide a specific list of required labels.
+      - '.*'
+  annotations:
+    # -- DEFAULT: disabled.
+    enabled: true | false
+    patterns:
+      # List of Go-style regular expressions used to filter desired annotations.
+      # Caution: The CloudZero system has a limit of 300 labels and annotations,
+      # so it is advisable to provide a specific list of required annotations.
+      - '.*'
+```
+
+### Upgrade Instructions
+
+If you have an existing CloudZero Agent deployment, follow these steps to upgrade:
+
+1. **Define the `values.yaml` Override Configuration**:
+   
+   Ensure your `values.yaml` override configuration includes the new settings outlined above. Note that some existing values may no longer be necessary.
+
+2. **Update the Helm Chart Repository**:
+   
+   ```sh
+   helm repo add cloudzero https://cloudzero.github.io/cloudzero-charts
+   helm repo update
+   ```
+
+3. **Upgrade the Deployment**:
+   
+   ```sh
+   helm upgrade --install <YOUR_RELEASE_NAME> -n <YOUR_NAMESPACE> cloudzero -f override-values.yaml
+   ```
+   
+   > **Replace** `<YOUR_RELEASE_NAME>` with the name you used to release the chart into your environment.
+   >
+   > **Replace** `<YOUR_NAMESPACE>` with the namespace you used for your deployment.
+
+### Deprecations and Breaking Changes
+
+1. **`node-exporter` Deprecation**:
+   
+   The `node-exporter` has been deprecated and is no longer used.
+
+2. **`kube-state-metrics` Configuration Changes**:
```

Comment:
Did we rename this `cloudzero-state-metrics`?

Reply:
Good eye! We renamed the service instance name (think alias for the actual name) which shouldn't be conflicting. However we do deploy it by default and name it `cloudzero-state-metrics`.  I think that should be captured!